client/gutenboarding/store/index.ts

```diff
@@ -10,11 +10,21 @@ import reducer, { State } from './reducer';
 import * as actions from './actions';
 import * as selectors from './selectors';
 
-const store = registerStore< State >( 'automattic/onboard', {
+export const STORE_KEY = 'automattic/onboard';
+
+registerStore< State >( STORE_KEY, {
 	reducer,
 	actions,
 	selectors,
 	// persist: [],
 } );
 
-export default store;
+export function useOnboardingState(): State {
+	return useSelect( select => select( STORE_KEY ).getState() );
+}
+
+export function useOnboardingDispatch() {
+	return useDispatch( STORE_KEY ) as {
+		[ D in keyof typeof actions ]: ( ...args: Parameters< typeof actions[ D ] > ) => void;
+	};
+}
```

Comment:
No string accessing the store (`wp.data.select( '???' )…`), just use these and get nice types 🎉

Reply:
:clap: